### ISSUE TITLE
Success message displayed in wrong place

### ISSUE THREAD

**Issue by jaydensmith** [verbb/formie#225]
**Description**
When using ajax submissions with "display a message" enabled, "hide form" off and the submission message position set to bottom of form, the success message is displayed in the button container.

![Screen Shot 2020-11-24 at 4 32 03 pm](https://user-images.githubusercontent.com/16344913/100052863-ea7ad500-2e72-11eb-83e8-e743d359840c.png)

![Screen Shot 2020-11-24 at 4 32 21 pm](https://user-images.githubusercontent.com/16344913/100052867-ee0e5c00-2e72-11eb-9362-60d0d2d2adb1.png)

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Comment by engram-design** [comment]
I can't exactly replicate this, because the button should be hidden along with the form? You're really not using custom templates?

But this is indeed an issue where the submission message isn't shown at all, because when set to the bottom of the form (above the submit button) that's not going to work because it's hidden!

Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.2.28",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.0](https://github.com/verbb/formie/releases/tag/1.3.0)
